spec/features/customers/add_item_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Add Item to Customer" do
```

Comment:
Since this feature is occuring on the Customer show page, we can simply add it to the `show_spec.rb` file as another describe block.